DSharpPlus/Entities/AutoModeration/RuleTriggerMetadata.cs

```diff
@@ -0,0 +1,67 @@
+using DSharpPlus.Enums;
+
+using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities;
+
+public sealed class RuleTriggerMetadata
```

Comment:
this should probably also be prefixed, something like DiscordAutoModerationRuleMetadata - it's long, but consistent